charts/akeyless-secure-remote-access/templates/_helpers.tpl

```diff
@@ -104,11 +104,11 @@ Checks persistent volume
 Get serviceAccountName
 */}}
 {{- define "akeyless-api-gw.getServiceAccountName" -}}
-    {{- if and (not .Values.sshConfig.service_account.serviceAccountName) ( not .Values.sshConfig.service_account.create )  }}
```

Comment:
please verify with @OriMankali if we need to support **backward computability** here

Reply:
resolved